test/cfg/windows.cpp

```diff
@@ -22,6 +22,22 @@
 #include <atlstr.h>
 #include <string>
 
+void invalidHandle_CreateFile(LPCWSTR lpFileName)
+{
+    HANDLE file = CreateFile(lpFileName, GENERIC_READ, 0, nullptr, CREATE_ALWAYS, FILE_ATTRIBUTE_NORMAL, nullptr);
```

Comment:
/home/runner/work/cppcheck/cppcheck/test/cfg/windows.cpp:27:42: debug: valueFlowConditionExpressions bailout: Skipping function due to incomplete variable GENERIC_READ [valueFlowBailoutIncompleteVar]
    HANDLE file = CreateFile(lpFileName, GENERIC_READ, 0, nullptr, CREATE_ALWAYS, FILE_ATTRIBUTE_NORMAL, nullptr);
                                         ^

Reply:
yes, the GENERIC_READ is not defined in the windows.cfg, will fix that